src/components/colorbar/draw.js

```diff
@@ -10,6 +10,7 @@
 'use strict';
 
 var d3 = require('d3');
+var tc = require('tinycolor2');
```

Comment:
scratch that comment :arrow_double_up: 

those `var tc` refer to tinycolor output, not ref to the tinycolor module. No change require.